sentry/src/main/java/io/sentry/ShutdownHookIntegration.java

```diff
@@ -32,7 +32,8 @@ public void register(final @NotNull IScopes scopes, final @NotNull SentryOptions
     Objects.requireNonNull(options, "SentryOptions is required");
 
     if (options.isEnableShutdownHook()) {
-      thread = new Thread(() -> scopes.flush(options.getFlushTimeoutMillis()));
+      thread =
+          new Thread(() -> scopes.flush(options.getFlushTimeoutMillis()), "sentry-shutdownhook");
```

Comment:
is there a change missing in this PR? There's no such overload for `flush`

Reply:
The added "sentry-shutdownhook" string is the name of the thread so it's easier to identify it, e.g. when debugging installed shutdown hooks.

There's no flush overload, instead it's: `new Thread(closure, "threadname")`

Reply:
of course, I couldn't spot the 2nd brace :D